python/sglang/srt/layers/radix_attention.py

```diff
@@ -50,7 +52,7 @@ def __init__(
         v_head_dim: int = -1,
         sliding_window_size: int = -1,
         is_cross_attention: bool = False,
-        quant_config: Optional[QuantizationConfig] = None,
+        quant_config: Optional["QuantizationConfig"] = None,
```

Comment:
`"` is not needed.
The reason is that you did not do `from __future__ import annotations`